src/wp-admin/includes/class-wp-debug-data.php

```diff
@@ -1413,16 +1378,16 @@ public static function debug_data() {
 			);
 		}
 
-		// Add more filesystem checks.
-		if ( defined( 'WPMU_PLUGIN_DIR' ) && is_dir( WPMU_PLUGIN_DIR ) ) {
-			$is_writable_wpmu_plugin_dir = wp_is_writable( WPMU_PLUGIN_DIR );
+		self::init_filters();
 
-			$info['wp-filesystem']['fields']['mu-plugins'] = array(
-				'label' => __( 'The must use plugins directory' ),
-				'value' => ( $is_writable_wpmu_plugin_dir ? __( 'Writable' ) : __( 'Not writable' ) ),
-				'debug' => ( $is_writable_wpmu_plugin_dir ? 'writable' : 'not writable' ),
-			);
-		}
+		/**
+		 * Fires before the debug information is gathered.
+		 *
+		 * This hook allows you to alter, add, or remove debug information before it is gathered.
+		 *
+		 * @since tbd
+		 */
+		do_action( 'pre_debug_information' );
```

Comment:
This action should not be introduced as part of this ticket.

Reply:
Agreed, will remove this.